tests/unit/test_precommit_hook.py

```
@@ -0,0 +1,151 @@
+        self._onboard(tmp_path)
+
+        with patch("slopmop.sm.main", return_value=1):
+            exit_code = cmd_hook(_hook_args("swab", tmp_path))
+
+        assert exit_code == 1
+
+
+class TestManifest:
+    """The .pre-commit-hooks.yaml manifest this repo exports."""
+
+    def _manifest(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_manifest`

(ANN202)